happytransformer/happy_bert.py

```diff
@@ -131,63 +136,72 @@ def __is_one_sentence(self, text):
                     sentence_found = True
                     break
         return True
+#-------------------------------------------------------#
+
+                # QUESTION ANSWERING #
+#-------------------------------------------------------#
+
+    def init_qa(self, model='bert-large-uncased-whole-word-masking-finetuned-squad'):
```

Comment:
This is a weird method. The only way to access it is through an instance of HappyBERT. Let's say I run the following.

```
happy_bert = HappyBERT()
happy_bert.init_qa()
```

Now the models and tokenizers for 'bert-base-uncased' and 'bert-large-uncased-whole-word-masking-finetuned-squad' are being stored in memory. This prohibitively and unecessarily increases the memory requirements of HappyTransformer.